core-services/image-mirroring/kueue/mapping_kueue

```diff
@@ -0,0 +1 @@
+registry.ci.openshift.org/openshift/kueue-controller:* quay.io/openshift/kueue-controller
```

Comment:
@ardaguclu @atiratree 

Is this standard? I'm not sure if we needed this for lws or jobset operands?

Reply:
Once we integrate lws and jobset operands with prow, I'd expect that these operands will be published in `pipeline:kubernetes-sigs-jobset` image in registry.ci.openshift.org. Why do you need this?